### PR TITLE
First recipe model

### LINKED ISSUE #6: Create JSON file with recipes
I need to create JSON object with recipes for later work

### PR DESCRIPTION
**Pull request by EwaTrzemzalska**
Resolves #6 
I've created model of recipe json file, that's how I see the keys for our recipe.

### CODE REVIEW

File: recipe.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "title": "your food's name",
+  "ingredients": [
+    "ingredient1",
+    "ingredient2",
+    "ingredient3"
+  ],
+  "instructions": [
```
Comment (smogg) [inline recipe.json]: Why are instructions an array?

File: recipe.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "your instruction goes here"
+  ],
+  "total calories": "your number of total calories goes here",
```
Comment (smogg) [inline recipe.json]: While this is technically a valid key, a common practice is to not use spaces in key names in JSON. Think of it as of variable name - this is what you're gonna be using in your code. `total_calories` is probably better

Comment (EwaTrzemzalska) [reply recipe.json]: You're right!

File: recipe.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "title": "your food's name",
+  "ingredients": [
+    "ingredient1",
+    "ingredient2",
+    "ingredient3"
+  ],
+  "instructions": "your instruction goes here",
+  "total_calories": "your number of total calories goes here",
+  "number_of_portions": "your number of portions goes here",
+  "calories_per_portion": "your calories will be calculated here"
```
Comment (smogg) [inline recipe.json]: Add some real life data here first

File: recipe.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  "total_calories": "2600",
```
Comment (smogg) [inline recipe.json]: What data type is the value supposed to be here?

Comment (EwaTrzemzalska) [reply recipe.json]: It should be number, not a string, fixed!

**Pull request merged**